query/language.peg

```diff
@@ -192,7 +192,7 @@ METRIC_NAME <- IDENTIFIER
 TAG_NAME <-    IDENTIFIER
 # TODO - may be refactored later.
 TIMESTAMP <-   NUMBER_NATURAL / STRING
-IDENTIFIER <-  "`" CHAR* "`" / !KEYWORD ID_SEGMENT ("." ID_SEGMENT)*
+IDENTIFIER <-  "`" CHAR* "`" / !(KEYWORD !ID_SEGMENT) ID_SEGMENT ("." ID_SEGMENT)*
```

Comment:
It should be `!ID_CONT` instead of `!ID_SEGMENT`, right?

Otherwise `"in3"` would be matched as a failing keyword, but `"inA"` would not.